src/System.Resources.Extensions/src/System.Resources.Extensions.csproj

```diff
@@ -1,8 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <Configurations>netstandard-Debug;netstandard-Release</Configurations>
+    <Configurations>netstandard-Debug;netstandard-Release;net461-Debug;net461-Release;netfx-Debug;netfx-Release</Configurations>
     <DefineConstants>$(DefineConstants);RESOURCES_EXTENSIONS;INTERNAL_NULLABLE_ATTRIBUTES</DefineConstants>
+    <DefineConstants Condition="'$(TargetsNetFx)' == 'true'">$(DefineConstants);netstandard;HIDE_READER</DefineConstants>
```

Comment:
Can we not do this?  Instead just use built in defines.  Make places that require both do `netstandard || NETFRAMEWORK` and just one use `NETFRAMEWORK`

Reply:
The HIDE_READER I would prefer to keep, the reason is that this is used in a common corelib file that gets compiled by other assemblies, so I wouldn't want to use a built in define that might affect how other consumers compile that file now. As for the netstandard one, sure I'll fix that to instead just modify the source.

Reply:
Actually I check the netstandard one and we have the same problem, the #ifdef is located at https://github.com/dotnet/runtime/blob/e755ce1e110decac70a121aafb3f3e467c5f1869/src/libraries/System.Private.CoreLib/src/System/IO/PinnedBufferMemoryStream.cs#L40-L44 so given it is in corelib and a file that multiple projects import I would be hesitant to change the ifdef to use a built in define which could have unintended consequences. I know defining netstandard for netfx is kind of ugly, but would yo be ok with me defining a different constant and changing that ifdef to instead have a `#if !netstandard || someOtherConstant`

Reply:
So I'm the one that added this file as shared and I don't expect it to be shared with other assemblies.  We don't expect many folks to share this source.

Our coding guidelines specify we use targeting properties rather than feature flags.
https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/project-guidelines.md#define-naming-convention